File: src/main/pokemon.py
import requests
from util import getAndCheck


def getPokemon() -> str | Exception:
    pokemon = input("Pokemon name: ").lower()
    url = f"https://pokeapi.co/api/v2/pokemon/{pokemon}"

    data = getAndCheck(url)

    height = int(data["height"]) / 10
    weight = int(data["weight"]) / 10

    return f"Height is {height} meter \nand it weighs {weight} kg"

File: src/kotlin/src/main/kotlin/pokemon.kt
package me.silent

import kotlinx.serialization.json.JsonObject

suspend fun getPokemon(): String {
    print("Pokemon name: ")
    val pokemon = readln()
    val url = "https://pokeapi.co/api/v2/pokemon/$pokemon"

    val data = getAndCheck(url) as JsonObject

    val height = data["height"].toString().toDouble() / 10
    val weight = data["weight"].toString().toDouble() / 10

    return "Height is $height meters\nand it weighs $weight kg"
}